src/test/kotlin/ReplicaSetDatabaseTests.kt

```diff
@@ -0,0 +1,59 @@
+import ReplicaSetDatabaseTests.Companion.testDb
+import com.moshbit.katerbase.MongoDatabase
+import com.moshbit.katerbase.equal
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+/**
+ * These tests require a replica set as [testDb]. Look for setup instructions in the "local-development" folder.
+ */
+class ReplicaSetDatabaseTests {
+
+  @Test
+  fun transactionTest() = runBlocking {
```

Comment:
```suggestion
  fun transactionSingleCollectionTest() = runBlocking {
```

Please also provide a test with a transaction that spans over multiple collections :)